cn_indian_payroll/cn_indian_payroll/overrides/loan_repayment.py

```
@@ -35,17 +24,12 @@ def before_save(self,method):
-
-                    
-                    
-
-                    
+                    date2 = f"{year}-{month:02d}-{last_day:02d}"                  
                     self.append("custom_loan_perquisite", {
                         "payment_date": i.payment_date,
                         "balance_amount":i.balance_loan_amount,
                         "perquisite_amount":round((i.balance_loan_amount*self.custom_loan_perquisite_rate_of_interest)/1200),
-                        "payroll_date":t2
+                        "payroll_date":date2
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)